res/sound/system/effects/FreeSoundEffects-dot-com License.txt

```diff
@@ -0,0 +1,21 @@
+https://www.freesoundeffects.com/licence.php
+
+License Agreement Explained.
+
+What You Can Do With Our Sounds:
+Add them to any feature film, video, website, Flash animation, podcast, instructional DVD, computer games....
+You can use our sounds for almost anything you can think of. No recurring royalties, no hidden costs, no submitting cue sheets.
+Add them to any software program, web application or computer game
+You just need to make sure the sounds are part of the app or game and not distributed as a separate sounds effects directory.
```

Comment:
You are violating this license in this PR/branch btw

Reply:
How so? The language explaining the licensing text seems unambiguous:

**What You Can Do With Our Sounds:**
Add them to any feature film, video, website, Flash animation, podcast, instructional DVD, computer games....
You can use our sounds for almost anything you can think of. No recurring royalties, no hidden costs, no submitting cue sheets.
Add them to any software program, web application or computer game
You just need to make sure the sounds are part of the app or game and not distributed as a separate sounds effects directory.
Use them in any theatre performance, art gallery, museum, performance piece
You can use these sounds in any type of live performance.

**What You Cannot Do With Our Sounds:**
You cannot resell these sounds as sound effects individually or in a collection
They can be incorporated into your project and the resulting product can be sold but not the sounds by themselves.
You cannot repurpose these sounds for resale as ringtones or any other sound effect type product
You cannot buy our sounds and then sell them as something else like a ringtone and you cannot combine our sounds and resell them as sound effects.
You cannot post them on a website for visitors to download.
You can use the sounds as background sounds for your website but you cannot post a link so that your visitors can download and use the sounds.
You cannot take our sounds and create an audio CD using only our sounds.
For example, you can use our Rain Forest Ambience to loop behind your yoga instructional video that includes a voice-over but you cannot create a Rain Forest Ambience audio CD using only our sound and resell them.

Reply:
„You just need to make sure the sounds are part of the app or game and not distributed as a separate sounds effects directory.“

Wouldn’t distributing source code fall under this (packaged being this)

Reply:
Ah, I think I maybe see now what you're keying on: "not distributed as a separate sounds effects directory".

I interpreted that based on the rest of their licensing text, which emphasizes that the sound effects files are not to be distributed separately _from the rest of the product_ (in our case, the LTR game). And there'd be no reason for us to ever do that. But it looks like you might be emphasizing "separate sound[s] effects directory."

In any case this, is easily remedied: I'll just move all audio files to a general ./res/sound/system/audio directory, and tweak the code so every sound reference looks in that directory. It's not as clean, but not a horrible defect.

Reply:
No I mean they shouldn't be distributed in the source code itself per license which is a problem for many open source projects. Distributed via packaged executable no problem, uploaded in git you'd violate that license.

Reply:
Temporary sound effects removed entirely.

Reply:
I can try to create some custom sound effects when I have some time.